frontend/packages/pipelines-plugin/src/components/repository/PipelineRunRepositories.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { PipelineRunKind } from '../../types';
+import { REPOSITORY_FIELDS, repositoryLabels } from './conts';
+
+export type PipelineRunRepositoriesProps = {
+  pipelineRun: PipelineRunKind;
+};
+
+const PipelineRunRepositories: React.FC<PipelineRunRepositoriesProps> = ({ pipelineRun }) => {
+  const { t } = useTranslation();
+  const plrLabels = pipelineRun.metadata.labels;
+  const repoLabel = repositoryLabels[REPOSITORY_FIELDS.REPOSITORY].labels[0];
+  if (!plrLabels || !plrLabels[repoLabel]) return null;
+
+  return (
+    <dl>
+      {Object.keys(repositoryLabels).map((repositoryLabel) => {
+        const labelKey = repositoryLabels[repositoryLabel].labels;
+        if (repositoryLabel === REPOSITORY_FIELDS.REPO_URL) {
+          return plrLabels[labelKey[0]] && plrLabels[labelKey[1]] ? (
```

Comment:
We could explicitly use the label here instead of using `labelKey[0]` and `labelKey[1]`, It will improve the readability.

Reply:
updated